aim/web/ui/src/components/kit/ExpressionAutoComplete/ExpressionAutoComplete.tsx

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+
+import CodeEditor from '@uiw/react-textarea-code-editor';
```

Comment:
Add `@uiw/react-textarea-code-editor` package and it's type declarations module to `package.json` file